l2geth/regenesis.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+./scripts/start.sh \
+    --datadir /mnt/2tb-ssd/kovan-regenesis/geth \
```

Comment:
I should be symlinking my mounted drive to something in `$HOME` so this script just works on other machines